docs/openapi-docs/src/main/scala/sttp/tapir/docs/openapi/EndpointToOpenAPIPaths.scala

```diff
@@ -108,7 +108,10 @@ private[openapi] class EndpointToOpenAPIPaths(schemas: Schemas, securitySchemes:
     EndpointInputToParameterConverter.from(header, Right(ASchema(ASchemaType.String)))
   private def cookieToParameter[T](cookie: EndpointInput.Cookie[T]) = EndpointInputToParameterConverter.from(cookie, schemas(cookie.codec))
   private def pathCaptureToParameter[T](p: EndpointInput.PathCapture[T]) = EndpointInputToParameterConverter.from(p, schemas(p.codec))
-  private def queryToParameter[T](query: EndpointInput.Query[T]) = EndpointInputToParameterConverter.from(query, schemas(query.codec))
+  private def queryToParameter[T](query: EndpointInput.Query[T]) = query.codec.format match {
```

Comment:
do I understand correctly that if the c-t is `text/plain`, then as it's the default one for query, we set the schema in `schema`, otherwise we need to specify a non-standard schema and that's why we use `content`?

some comment here would be necessary as to where this comes from; plus, maybe we can use a single method `EndpointInputToParameterConverter` - with an either? or maybe that wouldn't be too readable?

Reply:
I added a comment